numpy/distutils/command/build_ext.py

```diff
@@ -33,20 +33,31 @@ class build_ext (old_build_ext):
          "number of parallel jobs"),
         ('warn-error', None,
          "turn all warnings into errors (-Werror)"),
+        ('cpu-baseline=', None,
+         "specify a list of enabled baseline CPU optimizations"),
+        ('cpu-dispatch=', None,
+         "specify a list of dispatched CPU optimizations"),
+        ('disable-optimization', None,
+         "disable CPU optimized code(dispatch,simd,fast...)"),
+        ('simd-test=', None,
+         "specify a list of CPU optimizations to be tested against NumPy SIMD interface"),
     ]
```

Comment:
`simd-test` is new here and is not consistent with the other option definitions nor does it appear in lines 58-60

Reply:
it was belong to _simd module, wrong squash for one of fix up commits, move it to NPYV pr.